### PR TITLE
Create Gateways realm installation instructions

### PR DESCRIPTION
**Pull request by jsperhac**
## Description

Added markdown describing Gateways realm. This is documentation only and should accompany PR #1411.

## Motivation and Context

Provide brief documentation on, and instructions for installation of, this realm. 

## Tests performed

Markdown viewer. 

## Checklist:

- [X] The pull request description is suitable for a Changelog entry
- [X] The milestone is set correctly on the pull request
- [X] The appropriate labels have been added to the pull request

### CODE REVIEW

File: docs/gateways-realm.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Command:
+
+`etl_overseer.php -p gateways.bootstrap`
+
+Verify:
+
+Successful execution results in creation of the modw_gateways database schema with empty enduser, gateway, gatewayfact_by_day_joblist, and job_metadata tables.
+
+### b. Select and ingest jobs data for the modw_gateways database
+
```
Comment (jpwhite4) [inline docs/gateways-realm.md:29]:
```suggestion
XDMoD needs a mechanism to identify HPC jobs that were run via a gateway. The default mechanism is to use the last name associated with the user account that ran the jobs. Instructions for configuring the names are in the (User/PI Names guide)[user-names.md]. The steps to use this mechanism are:
1) Choose a suitable identifier to be used for a gateway (such as 'Gateway Proxy User')
1) Update the last name entries in `names.csv` for each gateway proxy user account.
1) Import the updated `names.csv`

**Comment by jpwhite4** [comment]
Also please can you update the command listing to match the documentation conventions:

https://open.xdmod.org/9.0/docs-conventions.html

**Comment by jpwhite4** [comment]
Also do we want a "Gateways realm (alpha)"  entry in the menu? I think yes.

File: docs/gateways-realm.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+## 1. Database configuration
+
+The gateways realm displays jobs information on science gateways for an Open XDMoD installation. It incorporates jobs that already exist in your Open XDMoD database. To do so, you will indicate the desired gateways jobs in the ingestion step indicated in b. below.
+
+### etl_overseer.php
+
+The database configuration steps are accomplished by running the script `etl_overseer.php` with different options. You should specify this script's full path when calling it (likely `/usr/share/xdmod/tools/etl/`, but check your system). This script accepts numerous parameters described in the help:
+
+`etl_overseer.php --help`
```
[the remark this answers is not in the document]
Comment (jsperhac) [reply docs/gateways-realm.md:14]: True, and it sounds as though /usr/share/xdmod/tools/etl/ is the RPM install path, so I will indicate that here.

File: docs/gateways-realm.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
+# Enabling the Gateways realm
+Open XDMoD 9.5
```
[the remark this answers is not in the document]
Comment (jsperhac) [reply docs/gateways-realm.md:2]: I will fix that.

File: docs/gateways-realm.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsperhac) [reply docs/gateways-realm.md:14]: ...and everywhere I refer to the command.

**Comment by jsperhac** [comment]
> Also do we want a "Gateways realm (alpha)" entry in the menu? I think yes.

Sounds good. I will add this under the "Installing" heading.

File: docs/gateways-realm.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
+# Enabling the Gateways realm
+
+The gateways realm displays jobs information due to science gateways. It does this by incorporating jobs data in a new XDMoD database, modw_gateways, created for capturing gateways data. This realm is not enabled by default in the current release. These instructions show you how to enable and populate the gateways realm in your Open XDMoD installation.
```
Comment (jpwhite4) [inline docs/gateways-realm.md:3]:
```suggestion
The gateways realm displays jobs information due to science gateways. It does this by incorporating jobs data in a new XDMoD database, `modw_gateways`, created for capturing gateways data. This realm is not enabled by default in the current release. These instructions show you how to enable and populate the gateways realm in your Open XDMoD installation.
```

File: docs/gateways-realm.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+```
+        "where": [
+            "p.last_name = '${community-user}'"
+```
+
+In its current state, the WHERE condition accepts as a parameter the gateway community user's surname. The command below specifies community user having last name 'Bunting'; replace this with an appropriate surname from your `modw.person` table.
+
+Command:
+
+`$ /usr/share/xdmod/tools/etl/etl_overseer.php -p gateways.ingest -d community-user=Bunting`
```
Comment (jpwhite4) [inline docs/gateways-realm.md:49]:
```suggestion
`$ /usr/share/xdmod/tools/etl/etl_overseer.php -p gateways.ingest -d community-user='Gateway proxy user'`
```

File: docs/gateways-realm.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+This step enables the UI to display the new Gateways realm that you configured in `resource_types.json`.
+
+Command:
+
+`$ acl-config`
+
+### 4. Restart services
+
+For good measure you may now restart all services (database, webserver, etc.).
```
Comment (jpwhite4) [inline docs/gateways-realm.md:121]: I don't think this step is necessary. If it were necessary then you should give instructions on how to do it.

**Review by jpwhite4 — CHANGES_REQUESTED** [review]

**Review by jpwhite4 — APPROVED** [review]

**Pull request merged**